suitcase/pizza_box/handlers.py

```diff
@@ -0,0 +1,123 @@
+import pandas as pd
+import os
+
```

Comment:
Let's add a description here (and we'll wait for answer from Chanaka and Dan below)
```py3
'''
These handlers read from data generated by the PIZZA boxes at NSLS-II.
There are currently two types: analog and encoder pizza boxes.

The analog pizza boxes have the form:
timestamp (s)    timestamp (ns)    index    value1 value2
where 
timestamp (s) : the time stamp in seconds
timestamp (ns) : the timestamp in ns
index: is a consecutively increasing integer
value1, [value2]:  a hexidecimal value that represents the digitized measured signal from the PIZZA box.
For the first version PIZZA box, there is only one value. For the second version, there are two values. 

The encoder pizza boxes have the form:
timestamp (s)    timestamp (ns)    value  index    value   di
timestamp (s) : the time stamp in seconds
timestamp (ns) : the timestamp in ns
value:  a hexidecimal value that represents the measured encoder reading from the PIZZA box.
index: is a consecutively increasing integer
di : unused quantity

More information about the Analog pizza box:
The analog PIZZA box has 8 channels, but not all are used.
There have currently been two versions:
Version 1: Only N channels were read. Each channel was written to a separate file.
Version 2: We can now read from 6 channels. 
However, to do this, the firmware had to be updated to 
write two channels per file. This explains the need for a 
second column for the analog pizza box file.

It might be possible that the analog PIZZA box firmware is 
updated to read from all 8 channels in the future.
'''

```

@lcdesilva can you confirm one question:
for version 1, how many channels max are supported? I think it's 2 but I want to confirm.

I will post an issue that this needs to be put into documentation. For the sake of time, I would rather get this in here at least, and you may discuss where this should go @danielballan .